device/linkstate/teprofile.yaml

```diff
@@ -0,0 +1,120 @@
+components:
+  schemas:
+    LinkState.TE:
```

Comment:
Following is just a question.
This TE profile is outside of ISIS protocol. It is directly under devices. Is it because the same TE profile can also attach to other protocol e.g. OSPFv2? If that is the goal, then could we have the TE profile configurable under each link (i.e. ethernet, VLAN interfaces) ? Usually TE properties are associated with an interface. Protocols just advertise that. What is the advantage doing it in this way?

Reply:
Thats kept as shared definition for ISIS and OSPF. But the TE object is instantiated under ISIS-Interfaces (you will see an references) and later under OSPF interfaces. You can also look through ReDocly to get better idea of where all the attributes hierarchically present.